src/iotest.f90

```diff
@@ -42,6 +44,19 @@ subroutine iotest
       end do
     end do
 
+    do a = 1, spn
+      do i = is, ie
+        do j = js, je
+          do k = ks, ke
+            spc(a,i,j,k) = 8.d0 + 1.d-2*i + 1.d-4*j + 1.d-6*k
```

Comment:
Can this be a function of `a` as well, just to be thorough?

Reply:
Done